app/node/categories/categoryCalculation.js

```diff
@@ -0,0 +1,21 @@
+const { search, exact } = require('./categoriesMap.json');
+
+const descriptionToCategory = Object.keys(exact).reduce((acc, category) => {
```

Comment:
This is classic for unit tests. I think we should start improving our test coverage and this is an easy example

Reply:
I don't know what you mean, but yes, I thought maybe we should write a test to validate the JSON structure, make sure there are no duplicates.

Reply:
What I mean is that this code is self contained and can be tested easily in a unit test.

Given a json categories map, a test for exact matches and a test for search matches. We can discuss offline if my meaning is not clear